packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -783,6 +798,8 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
 
     this.serverlessV2MaxCapacity = props.serverlessV2MaxCapacity ?? 2;
     this.serverlessV2MinCapacity = props.serverlessV2MinCapacity ?? 0.5;
+    this.secondsUntilAutoPause = props.secondsUntilAutoPause?.toSeconds();
```

Comment:
If `props.secondsUntilAutoPause` is `undefined` will `this.secondsUntilAutoPause` be set to `undefined` or 0? I ask because in the ServerlessCluster version of this prop they check if its 0 and then set it to undefined [here](https://github.com/aws/aws-cdk/blob/a2cd7ae80168610db2df515a11e2a68571c662fb/packages/aws-cdk-lib/aws-rds/lib/serverless-cluster.ts#L505).